File: correcao_lista_07_03/ex_3.py
temp = []
media = [0]*3

clear

temp.append([0] * 7)
temp.append([0] * 7)
temp.append([0] * 7)

for li in range(0,3):
    for co in range(0,7):
        temp[li][co] = float(input(f"Digite o valor do dia {co} da semana {li}:  "))

for li in range(0,3):
    media[li] = sum(temp[li]) / len(temp[li])
    for co in range(0,7):
        if (temp[li][co] < media[li]):
          print(f" Temperatura do dia {co} - Semana {li}, {temp[li][co]}, menor que a média {media[li]}"); 

File: correcao_lista_07_03/ex_3.c
#include <stdio.h>

void main(){
    float temp[3][7];
    int li, co;

    for(li = 0; li <3; li++){
        for(co = 0; co < 7; co++){
            printf("Digite o valor do dia %d da semana %d: ", (co+1), (li+1));
            scanf("%f", &temp[li][co]);    
        }
    }

    for(li = 0; li <3; li++){
        for(co = 0; co < 7; co++){
            printf(" Temperatura do dia %d - Semana %d : %.2f\n", (co+1), (li+1), temp[li][co]); 
        }
    }
}